integration/templates/cert_manager_values.go

```diff
@@ -1,18 +0,0 @@
-// +build k8srequired
-
-package templates
-
-// CertManagerValues values required by cert-manager-app chart.
-const CertManagerValues = `---
-name: cert-manager
-namespace: kube-system
-
-userID: 1000
-groupID: 1000
-
-image:
-  registry: quay.io
-  name: giantswarm/cert-manager-controller
-  tag: v0.9.0
-e2e: true
```

Comment:
all the other values are values.yaml, except e2e which we used before to define/include priority class template in e2e tests only, but this is no longer used, so wiped the whole file and references to it